docs/extensions/overview.md

```diff
@@ -2,32 +2,95 @@
 layout: docu
 title: Extensions
 ---
-DuckDB has a number of extensions available for use. Not all of them are included by default in every distribution, but DuckDB has a mechanism that allows for remote installation.
 
-## Remote Installation
+## Overview
 
-If a given extensions is not available with your distribution, you can do the following to make it available.
+DuckDB has a flexible extension mechanism that allows for dynamically loading extension.
+These may extend DuckDB's functionality by providing support for additional file formats, introducing new types, and domain-specific functionality.
+
+> Extensions are loadable on all clients (e.g., Python and R).
+> While extensions can restrict their target platforms, the extensions distributed by the DuckDB developer team support all major platforms (Linux, MacOS, Windows).
```

Comment:
While this is broadly speaking true, we don't presently support linux_arm64 or windows_amd64_mingw for example

Reply:
Extension can restrict their target platforms, for example `jemalloc` wraps an improved malloc implementation, but the original C++ code is not compatible with OSX malloc implementation.
??

Reply:
Not sure jemalloc is a useful example there - it's only ever linked in, and only on linux

Reply:
Then something more factual like:
```
Extension distributed via the official repository are built and tested on MacOS (both arm64 and amd64), Windows (x and y) and Linux (a, b, c, and d).
```

Reply:
And then in the "working with extensions" page one can go more in-depth about details.

Reply:
I pushed a clarification